src/index.html

```diff
@@ -4,9 +4,10 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <link rel="icon" href="https://cdn-icons-png.flaticon.com/512/2785/2785819.png" type="image/png" />
+    <title>Covid Dashboard</title>
   </head>
   <body>

```

Comment:
Also we should add an empty "root" div which represents an entry-point block of our app. You will find an example of usage in our **main** JS file. 
```suggestion
  <body>
    <div id="root"></div>
```